analytic_base_department/__manifest__.py

```diff
@@ -4,7 +4,7 @@
 {
     "name": "Base Analytic Department Categorization",
     "summary": "Add relationshet between Analytic and Department",
-    "version": "13.0.1.1.0",
+    "version": "13.0.1.2.0",
```

Comment:
FYI No need to manually bump version, this is done by the merge bot

Reply:
Good to know! I specifically went through the [guidelines](https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#version-numbers) to refresh my memory and I didn't see any mention about the merge bot.

Should I keep it as it is or undo this change?